YOLO/ViewController.swift

```diff
@@ -558,7 +596,16 @@ extension ViewController: AVCapturePhotoCaptureDelegate {
         if let dataImage = photo.fileDataRepresentation() {
             let dataProvider = CGDataProvider(data: dataImage as CFData)
             let cgImageRef: CGImage! = CGImage(jpegDataProviderSource: dataProvider!, decode: nil, shouldInterpolate: true, intent: .defaultIntent)
-            let image = UIImage(cgImage: cgImageRef, scale: 0.5, orientation: UIImage.Orientation.right)
+            var orientation = UIImage.Orientation.right
```

Comment:
Am I wrong or is this change related to landscape mode? Maybe we need to merge the branch landscape first! @john-rocky What do you think about it?

Reply:
This is indeed code that supports landscape mode, and it will still work in portrait mode, but I can change it back to the original code if necessary, or I can commit landscape mode to reflect that.